lib/evmone/vm.cpp

```diff
@@ -59,12 +59,12 @@ evmc_set_option_result set_option(evmc_vm* c_vm, char const* c_name,
         vm.add_tracer(create_instruction_tracer(std::clog));
         return EVMC_SET_OPTION_SUCCESS;
     }
-    else if (name == "trace-bin")
+    if (name == "trace-bin")
```

Comment:
In any given invocation `name` has only one value, so there does not seem to be a point in removing the `else`?

Reply:
Yeah, my fault, it is already worked. Fixed, thanks.